commands/sketch/new.go

```diff
@@ -55,19 +55,32 @@ func NewSketch(ctx context.Context, req *rpc.NewSketchRequest) (*rpc.NewSketchRe
 		return nil, err
 	}
 
+	templateDir := configuration.Settings.GetString("directories.template")
```

Comment:
I would not use `directories.template`, for two reasons:
* the prefix "directory." refers to places were there may be multiple things inside, but the sketch template is only one
* "template" is a bit too generic

IMHO is better `sketch.template`.